core-bundle/contao/templates/_new/component/_table.html.twig

```diff
@@ -124,7 +124,7 @@
                                     <td>
                                         {%- block table_body_cell_content %}
                                             {%- block table_cell_content %}
-                                                {{- cell|nl2br|insert_tag_raw -}}
+                                                {{- cell|insert_tag_raw -}}
```

Comment:
Should we maybe also remove the insert tag filter by default?
```suggestion
                                                {{- cell -}}
```

Reply:
Then we would have to remove it from the `caption` element as well, wouldn‘t we?

Reply:
Removed in 5e25a594420ec236b4b98f682612a3875745647e.